packages/common/src/allo/transaction-sender.ts

```diff
@@ -185,24 +200,46 @@ export function createMockTransactionSender(): TransactionSender & {
       return {
         transactionHash: txHash,
         blockHash: `0x${Math.random().toString(16).slice(2)}` as Hex,
-        blockNumber: 1n,
+        blockNumber: BigInt(1),
         logs: [],
       };
     },
+
+    async address(): Promise<Address> {
+      return ("0x" + "0".repeat(40)) as Address;
+    },
   };
 }
 
+export async function sendRawTransaction(
+  sender: TransactionSender,
+  args: TransactionData
+): Promise<Result<Hex>> {
+  try {
+    const tx = await sender.send({
+      to: args.to,
+      data: args.data,
+      value: args.value,
+    });
+    return success(tx);
+  } catch (err) {
+    return error(
+      new AlloError(`Failed to send raw transaction: ${String(err)}`, err)
+    );
+  }
+}
+
 export async function sendTransaction(
   sender: TransactionSender,
-  args: Parameters<typeof encodeFunctionData>[0] & { address: Address }
+  args: EncodeFunctionDataParameters<Abi, string> & { address: Address }
 ): Promise<Result<Hex>> {
   try {
     const data = encodeFunctionData(args);
 
     const tx = await sender.send({
       to: args.address,
       data: data,
-      value: 0n,
+      value: BigInt(0),
```

Comment:
bigint literals should be supported, if not then the build config is wrong, we should fix that instead